### ISSUE TITLE
How to make Electron app launch x,000ms faster

### ISSUE THREAD

**Issue by bm777** [saltyshiomix/nextron#410]
@saltyshiomix 

How can I make electrons faster in production? The build app takes around 5-12 seconds to load.

I discovered how atom.io has a launch time faster. They used v8-snapshot with electron-linker following this [guide](https://blog.inkdrop.app/how-to-make-your-electron-app-launch-1000ms-faster-32ce1e0bb52c). How can I have v8-snapshot running for the module I'm using in my Nextron app?

I'm asking if you have already guidance for Nextron. 

(I'm fearing to waste time if you have already done that, it could take me more time because it will be my first time.)

I'm amazed by the work v8-snapshots in an Electron app.

<img width="721" alt="Screenshot 2023-10-04 at 14 29 58" src="https://github.com/saltyshiomix/nextron/assets/29865600/4f727d12-be25-4c57-adfc-02978af46b46">

**Comment by FranciscoJBrito** [comment]
Hi @bm777 👋🏻, I would like to contribute to this improvement, have you already made progress on this?

**Comment by bm777** [comment]
@FranciscoJBrito, yes definitely.

I will create an empty Nextron app with Tailwindcss + JavaScript.

And we can work on it. It will be in another branch of this repo. Will provide the name in a few minutes.

**Comment by bm777** [comment]
@saltyshiomix Could we work on another (new) in your repo?

**Comment by bm777** [comment]
@FranciscoJBrito This is the template [js + tailwindcss nextron](https://github.com/bm777/nextron-speed-launch) we can use to configure v8-snapshots.

This is the [reference](https://blog.inkdrop.app/how-to-make-your-electron-app-launch-1000ms-faster-32ce1e0bb52c).

**Comment by FranciscoJBrito** [comment]
@bm777 Awesome, I'll be working on [this branch](https://github.com/bm777/nextron-speed-launch) then 🙌🏻

**Comment by bm777** [comment]
You can create a branch francis if you want :)

**Comment by saltyshiomix** [comment]
That's so interasting!

@bm777 
> Could we work on another (new) in your repo?
Sure!!!

**Comment by bm777** [comment]
Cool. I'm not allowed to push code to another branch. Maybe you can create a branch for us 😃 
@saltyshiomix

**Comment by saltyshiomix** [comment]
@bm777 @FranciscoJBrito 

I just added you as a collaborator: https://github.com/saltyshiomix/nextron/invitations

Feel free to make a branch or create PR :)

**Comment by bfan1256** [comment]
Hello! We are big fans of nextron. By chance, what is the status of this? Or does anyone have guides on how to use this? 

Brandon

**Comment by bm777** [comment]
Hi @bfan1256 I will work on it in the 1 week of 2024.  And give you some updates.

**Comment by bm777** [comment]
Do you mean to use v8 in Nextron?

**Comment by rashid-trova** [comment]
Yes, how i can use v8 snapshots in Nextron, I am experiencing a loading delay on my Nextron App. or is there any way to improve its loading?
